src/preact-render-spy.js

```diff
@@ -270,11 +276,19 @@ class FindWrapper {
     );
   }
 
+  name() {
+    verifyFoundNodes(this);
+    verifyOnlySingleNode(this);
+
+    const nodeName = this[0].nodeName;
+    return typeof nodeName === 'function'
```

Comment:
Please don't return the result of a ternary, I'd generally prefer to see 

```
if (typeof nodeName === 'function') {
  return ...;
}
return ...;
```

But that might just be my old jQuery style preferences, I think it's easier to understand if's than ternaries 90% of the time as a reader.

Reply:
Also, we need to be aware of the `displayName`: 
* `func.displayName` should be used if it exists, like `func.displayName || func.name` I think.

Reply:
^ correct

Reply:
For some reason I thought preact doesn't support `displayName`. I will fix that :)